app/components/Search.js

```diff
@@ -101,34 +92,27 @@ class Search extends Component {
           </TouchableHighlight>
 
           <ScrollView>
```

Comment:
I'm glad that you were able to make ScrollView work. I think ListView could have worked, but this makes sense.